src/DacpacTool/Extensions.cs

```diff
@@ -154,6 +154,8 @@ public static void AddReference(this TSqlModel model, string referencePath, stri
                 }
             }
 
+            File.Copy(referencePath, Path.Combine(Path.GetDirectoryName(typeof(PackageBuilder).Assembly.Location), Path.GetFileName(referencePath)), true);
```

Comment:
I'm wondering if rather than copying the referenced `.dacpac` to the location of the tool we should instead copy it next to the output `.dacpac` we are building (which I believe we are already doing) and execute the tool with the working directory set to the output folder?

Reply:
Yeah, the test failure inidcates that it is a better solution. Problem is that in many of the tests we do not even have an output .dacpac...

Reply:
I think we can solve this within the MSBuild part and that the tool doesn't necessarily have to worry about it.

Reply:
The `<Exec>` task we are using has a `WorkingDirectory` property that we are currently not setting, meaning it will run the command in the folder containing the `.csproj`. I think we can change that to be the output folder.

Reply:
Feel free to update this PR !

Reply:
Did some experimenting, but now the problem becomes that the set of input files are relative to the project folder so now it is failing on adding those. Might need to think about this some more before we can fix this.